docs/api.rst

```diff
@@ -572,10 +605,10 @@ mergeMapConcurently
 
   mergeMapConcurently :: (a -> Stream b) -> int -> Stream a -> Stream b
 
-Lazily applies a function ``f`` to each event on a stream, merging them into the
+Lazily apply a function ``f`` to each event in a stream, merging them into the
 resulting stream at the specified concurrency. Once concurrency number of streams
 are being merged, newly arriving streams will be merged after an existing one
-ends.::
+ends. ::
```

Comment:
Hmmm, this paragraph now mixes soft and hard returns.  I think we just need to pick one and go with it.  As mentioned above, I prefer relying on soft wrap, but won't die on a hill for it if there's a compelling reason to use hard returns everywhere.